lib/capybara/cuprite/driver.rb

```diff
@@ -33,7 +33,7 @@ def initialize(app, options = {})
         @screen_size = @options.delete(:screen_size)
         @screen_size ||= DEFAULT_MAXIMIZE_SCREEN_SIZE
 
-        @options[:save_path] = Capybara.save_path.to_s if Capybara.save_path
+        @options[:save_path] ||= Capybara.save_path.to_s if Capybara.save_path
```

Comment:
Don't know if you thought about it but I would have liked the option to be called download_path, maybe also rename it in ferrum. Probably a decisions for @route